src/stdatamodels/model_base.py

```diff
@@ -570,7 +588,24 @@ def save(self, path, dir_path=None, *args, **kwargs):
     @staticmethod
     def open_asdf(init=None, ignore_unrecognized_tag=False, **kwargs):
         """
-        Open an asdf object from a filename or create a new asdf object
+        Open an asdf object from a filename or create a new asdf object.
+
+        Parameters
+        ----------
+        init : str, file object, `~asdf.AsdfFile`, dict
+            - str : file path: initialize from the given file
+            - readable file object: Initialize from the given file object
+            - `~asdf.AsdfFile` : Initialize from the given`~asdf.AsdfFile`.
+            - dict : Initialize from the given dictionary.
+        ignore_unrecognized_tag : bool
+            If `True`, ignore tags that are not recognized.
+        **kwargs : dict
```

Comment:
I learned today that `args` and `kwargs` should be documented with the `*`s but without a type. See the end of this section:
https://numpydoc.readthedocs.io/en/latest/format.html#parameters

This is in many places, and I have definitely been doing this wrong in all the jwst code, so it's not critical to fix it here either, just noting it.

```suggestion
        **kwargs
```

Reply:
I went ahead and fixed them because it's an easy and pretty safe "replace-all"